### ISSUE TITLE
Error: Unsupported top level event type "topScrollEndMomentum" dispatched

### ISSUE THREAD

**Issue by sivakusayan** [microsoft/react-native-windows#12272]
### Problem Description

When running [ScrollViewSnapExample](https://github.com/microsoft/react-native-windows/blob/main/packages/playground/Samples/scrollViewSnapSample.tsx) in the playground, we get the following errors by scrolling the container:

```
Error: Unsupported top level event type "topScrollBeginMomentum" dispatched
Error: Unsupported top level event type "topScrollEndMomentum" dispatched
```

### Steps To Reproduce

1. Launch the [react-native-windows playground](https://github.com/microsoft/react-native-windows/tree/main/packages/playground).
2. Load `scrollViewSnapExample.tsx`
3. Start scrolling the container.

### Expected Results

We don't throw errors

### CLI version

12.0.0-alpha.15

### Environment

```markdown
System:
  OS: Windows 10 10.0.19045
  CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11850H @ 2.50GHz
  Memory: 19.15 GB / 31.73 GB
Binaries:
  Node:
    version: 18.18.0
    path: C:\Program Files\nodejs\node.EXE
  Yarn:
    version: 1.22.19
    path: ~\AppData\Roaming\npm\yarn.CMD
  npm:
    version: 9.8.1
    path: C:\Program Files\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK: Not Found
  Windows SDK:
    AllowDevelopmentWithoutDevLicense: Enabled
    AllowAllTrustedApps: Enabled
    Versions:
      - 10.0.18362.0
      - 10.0.19041.0
      - 10.0.22621.0
IDEs:
  Android Studio: Not Found
  Visual Studio:
    - 17.7.34031.279 (Visual Studio Professional 2022)
Languages:
  Java: Not Found
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react: Not Found
  react-native: Not Found
  react-native-windows: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: Not found
  newArchEnabled: Not found
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found
```

### Target Platform Version

10.0.19041

### Target Device(s)

Desktop

### Visual Studio Version

Visual Studio 2022

### Build Configuration

Debug

### Snack, code example, screenshot, or link to a repository

The code example is in the playground exampled linked above.

**Comment by sivakusayan** [comment]
I think the issue is that the code emits the `topScrollBeginMomentum` and `topScrollEndMomentum` events in [`ScrollViewShadowNode::AddHandlers`](https://github.com/microsoft/react-native-windows/blob/694d71951c61265d3380262362f4d689806d75af/vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp#L238), but should really be emitting [`topMomentumScrollBegin` and `topMomentumScrollEnd`](https://github.com/microsoft/react-native-windows/blob/694d71951c61265d3380262362f4d689806d75af/vnext/Microsoft.ReactNative/Views/ScrollViewManager.cpp#L456). I might also be misunderstanding something, though.

**Comment by lukutism** [comment]
@sivakusayan Thanks, fixed issue #12285
